apps/frontend/src/app/access-portal/pages/home/components/post-card/post-card.component.html

```diff
@@ -1,11 +1,16 @@
 <div class="border-neutral-6 flex flex-col rounded-sm border-4 p-2 sm:p-7">
   <div class="flex justify-between">
     <div class="flex items-center">
-      <img
-        [src]="sanitizeUrl(publication.profile.picture?.original?.url)"
-        alt="avatar example"
-        class="h-12 w-12 rounded-full object-cover"
-      />
+      <button
```

Comment:
![image](https://github.com/Venture-Miner/wagademy/assets/111710515/4facbb0d-6154-48f3-a423-bb2cff4bbd1d)

it would be better if there was some horizontal padding while on small devices